### DISCUSSION TITLE
La base de datos no carga en producción

### DISCUSSION THREAD

**Discussion by JARS2009** [NativePHP/laravel#369] [Q&A]
**Al momento de ejecutar "php artisan native:serve" mi lista de empleados carga de manera normal** 
![Captura de pantalla 2024-09-11 182025](https://github.com/user-attachments/assets/6211affc-b849-458a-ba33-14f158d9bb0b)

**Pero luego al compilar y instalar en mi pc en la misma vista tengo este error**
![Captura de pantalla 2024-09-11 181918](https://github.com/user-attachments/assets/230257ec-de3f-4512-901f-cea32dfa7223)

**este es mi método index:**
![Captura de pantalla 2024-09-11 185952](https://github.com/user-attachments/assets/e93d5ae8-91b4-4067-b2c2-421234c4f27c)

**Mi model:**
![image](https://github.com/user-attachments/assets/7cac293b-463a-4f12-99ba-99412178e89f)

**Comment by simonhamp** [comment]
When running builds, your application environment is switched to `production` automatically, which makes debugging these issues slightly trickier.

But [there should be logs](https://nativephp.com/docs/1/getting-started/debugging#check-the-logs) stored in the "Appdata" folder. If you can dig these out, they may help you to identify the root cause of this issue.

Please report back what you find as I've seen this issue quite a few times and it makes me think we're missing something in the build step.
